### DISCUSSION TITLE
Azure oauth access problem

### DISCUSSION THREAD

**Discussion by alms1** [outline/outline#5904] [Self Hosting]
I just installed the service in an app service. When I add authentication with Azure AD, I get the following error. Any ideas?

![image](https://github.com/outline/outline/assets/27727237/94442681-714d-40bf-a7e5-0fccf6755f43)

**Comment by tommoor** [comment]
Post the server logs, there will be more detail available there

**Reply by alms1** [reply]
I have no server logs. I just created the service with Azure Outh, but it seems that my account does not have permissions to access the wiki. Is there any additional configuration?

![image](https://github.com/outline/outline/assets/27727237/dc48eca2-e235-45ef-bca4-265820d5629d)

**Reply by tommoor** [reply]
Without posting the server logs it's not going to be possible to debug further I'm afraid
